_data/people_list.yml

```diff
@@ -0,0 +1,618 @@
+
+- name: Raul Epure
+  affiliation: University of Kaiserslautern-Landau
+  status: retired
+  github: raulepure
+  comment: Used to work in TU KL, under whom and for what ? who ?
+
+- name: Rafael Fourquet
+  affiliation: University of Kaiserslautern-Landau
+  status: retired
+  comment: General infrastructure who ?
+  github: rfourquet
+
+- name: Sebastian Gutsche
+  affiliation: University of Kaiserslautern-Landau
+  website: https://sebasguts.github.io/
+  status: retired
+  github: sebasguts
+
+- name: William Hart
+  affiliation: University of Kaiserslautern-Landau
+  status: retired
+  github: wbhart
+  comment: Initial contributor to Oscar.
+
+- name: Florian Heiderich
+  affiliation: University of Siegen
+  website: https://www.heiderich.org/math/
+  status: retired
+  github: heiderich
+
+- name: Fredrik Johansson
+  affiliation: Institut de Mathématiques de Bordeaux
+  website: http://fredrikj.net/
+  email: fredrik.johansson@gmail.com
+  github: fredrik-johansson
+  status: retired
+  comment: The FLINT guy.
+
+- name: Marek Kaluba
+  affiliation: KIT Karlsruhe
+  website: https://kalmarek.github.io/
+  email: kalamar@mailbox.org
+  github: kalmarek
+  status: retired
+  comment: Freelance mathematician, formerly at KIT
+
+- name: Avinash (Avi) Kulkarni
+  affiliation: Dartmouth College
+  website: https://math.dartmouth.edu/~akulkarn/index.html
+  email: avinash.a.kulkarni@dartmouth.edu
+  github: a-kulkarn
+  status: retired
+  comment: Who ?
+
+- name: Frank Lübeck
+  affiliation: RWTH Aachen University
+  website: http://www.math.rwth-aachen.de/~Frank.Luebeck/index.html
+  email: Frank.Luebeck@math.rwth-aachen.de
+  github: frankluebeck
+  status: retired
+
+- name: Alexander Mattes
+  affiliation: University of Kaiserslautern-Landau
+  website: https://github.com/alexandermattes
+  github: alexandermattes
+  status: retired
+  comment: Masters student of Claus Fieker
+
+- name: Sachin Mohan
+  affiliation: University of Kaiserslautern-Landau
+  website: https://sachinkmohan.github.io/
+  email: sachinkmohan@proton.me
+  github: sachinkm308
+  status: retired
+  comment: Who ?
+
+- name: Oleksandr Motsak
+  affiliation: IMAGINARY
+  status: retired
+  github: malex984
+  comment: Who ?
+
+- name: Markus Pfeiffer
+  affiliation: St Andrews
+  github: markuspf
+  status: retired
+
+- name: Delphine Pol
+  affiliation: University of Kaiserslautern-Landau
+  status: retired
+  github: delphinepol
+  comment: Used to be a postdoc at Kaiserslautern, until ~2020
+
+- name: Martin Raum
+  affiliation: Chalmers University
+  website: https://www.raum-brothers.eu/martin/
+  email: raum@chalmers.se
+  github: martinra
+  status: retired
+
+- name: Daniel Schultz
+  affiliation: University of Kaiserslautern-Landau
+  status: retired
+  github: tthsqe12
+  comment: who ?
+
+- name: Carlo Sircana
+  affiliation: University of Kaiserslautern-Landau
+  status: retired
+  comment: PhD student of Claus Fieker
+  github: CarloSircana
```

Comment:
Will your script also put these comments in a fixed order?

And sort the entries in this file overall?

Reply:
* The fields in each yml entry are sorted alphabetically. So "comment" before "github".
* The yml entries themselves are sorted (by the python script) according to last name of contributor.